src/components/Home/MostViews/index.test.tsx

```diff
@@ -0,0 +1,8 @@
+import testRender from '../../../../.jest/testRender'
+import { render } from '@testing-library/react'
+import React from 'react'
+import MostViews from '.'
+
+describe('components/Home/MostViews', () => {
+  testRender(<MostViews />)
```

Comment:
works, but will lead to unreliable tests as by doing it this way, all the real requests are actually being fired, meaning the views endpoint and another `queryMetadata()` call. Ideally, both are mocked inside this file, also to test all possible return types. Could be good exercise for tomorrow's testing intro

Reply:
and also spills a console warning which should be addressed https://github.com/oceanprotocol/market/actions/runs/3413734266/jobs/5680799611#step:6:297

Reply:
on myside there are multiple errors on the tests, didn't really understood the point. I'll look into mocking the call.